c_common/models/fec_models.mk

```diff
@@ -0,0 +1,7 @@
+MAKEFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
```

Comment:
by changing the order of how the main make here works, you could remove this 
+include $(CURRENT_DIR)../front_end_common_lib/local.mk
to point at the spinnaker_tools version.
aka.a build lib, install lib, build models, install models.

Reply:
Done